spark3-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DynamicFileFilterExec.scala

```diff
@@ -47,13 +49,45 @@ case class DynamicFileFilterExec(
   override protected def doExecute(): RDD[InternalRow] = scanExec.execute()
   override protected def doExecuteColumnar(): RDD[ColumnarBatch] = scanExec.executeColumnar()
 
+  protected def doPrepare(): Unit
```

Comment:
Is this needed? I thought this was defined by `SparkPlan`

Reply:
Not needed Ryan. removing it.